packages/esm-patient-conditions-app/src/conditions/conditions-widget.component.tsx

```diff
@@ -29,10 +31,15 @@ import {
   useConditions,
   useConditionsSearch,
 } from './conditions.resource';
-import { type ConditionSchema } from './conditions-form.workspace';
 import styles from './conditions-form.scss';
 
 interface ConditionsWidgetProps {
+  schema: z.ZodObject<{
+    abatementDateTime: z.ZodNullable<z.ZodOptional<z.ZodDate>>;
+    clinicalStatus: z.ZodEffects<z.ZodString, string, string>;
+    conditionName: z.ZodEffects<z.ZodString, string, string>;
+    onsetDateTime: z.ZodNullable<z.ZodDate>;
+  }>;
```

Comment:
I feel that there's a better annotation than this or that can be inferred from usage.